tree/color.go

```diff
@@ -21,14 +23,5 @@ var (
 )
 
 func randomColor() *color.Color {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Seeding should be done exactly 1 time before using rand. Here Seed can be called multiple times, also in leaf.go. So I moved rand.Seed to init() func of leaf.go.

Reply:
I want to random color every time, are you sure it work?

Reply:
It work.

Reply:
That's the point, documentation says that we need to seed only 1 time.